app/dashboard/migrations/0097_auto_20180717_0151.py

```diff
@@ -0,0 +1,78 @@
+# Generated by Django 2.0.7 on 2018-07-17 01:51
+
+from django.db import migrations
+from dashboard.utils import get_web3
+from django.conf import settings
+
+
+def get_profile(Profile, username):
+    profiles = Profile.objects.filter(handle__iexact=username)
+    if profiles.exists():
+        return profiles.first()
+    return None
+
+
+def save_address(profile, address):
+    profile.preferred_payout_address = address
+    profile.save()
+
+
+def try_to_link_address_to_profile(Profile, username, address):
+    if not username or not address:
+        return False
+    profile = get_profile(Profile, username)
+    if profile:
+        save_address(profile, address)
+        #print('-')
+        return True
+    return False
+
+
+def backwards_func(apps, schema_editor):
+    pass
```

Comment:
Do we want to handle rollbacks here?

Reply:
i think the empty backwards fund is fine bc the previous migration will destroy the table with the data created in this migration